spec/system/admin/products_v3/products_spec.rb

```diff
@@ -11,6 +11,10 @@
     login_as_admin
   end
 
+  let(:producer_search_selector) { 'input[placeholder="Search for producers"]' }
+  let(:categories_search_selector) { 'input[placeholder="Search for categories"]' }
+  let(:tax_categories_search_selector) { 'input[placeholder="Search for tax categories"]' }
+
   it "can see the new product page" do
```

Comment:
This spec file is getting huge now. How about we split the the relevant specs into different files? Like products_{feature_name}_specs etc?